tests/src/GC/Stress/Framework/LoaderClass.cs

```diff
@@ -126,6 +126,102 @@ public void Load(string assemblyName, string[] paths, ReliabilityFramework rf)
         }
     }
 
+    /// <summary>
+    /// Checks the test's entry point for an STA or MTA thread attribute.  Returns
+    /// the apartment state if set, or ApartmentState.Unknown
+    /// </summary>
+    /// <returns></returns>
+    public ApartmentState CheckMainForThreadType()
+    {
+        if (assem != null)
+        {
+            MethodInfo mi = assem.EntryPoint;
+            object[] attrs = mi.GetCustomAttributes(typeof(STAThreadAttribute), false);
+            if (attrs != null && attrs.Length > 0)
+            {
+                return (ApartmentState.STA);
+            }
+            attrs = mi.GetCustomAttributes(typeof(MTAThreadAttribute), false);
+            if (attrs != null && attrs.Length > 0)
+            {
+                return (ApartmentState.MTA);
+            }
+        }
+        return (ApartmentState.Unknown);
+    }
+#else
+    /// <summary>
+    /// Executes a LoadFrom in the app domain LoaderClass has been loaded into.  Attempts to load a given assembly, looking in the
+    /// given paths & the current directory.
+    /// </summary>
+    /// <param name="path">The assembly to load</param>
+    /// <param name="paths">Paths to search for the given assembly</param>
+    public void LoadFrom(string path, string[] paths, object rf)
+    {
```

Comment:
so I see there's a lot of duplicated code and I am not asking you to spend more time to not duplicate it. but would you mind adding a one line comment just to say what the difference is between this and AD for each of the duplicated code block?

Reply:
I wonder, should we instead delete all the AppDomain related stuff from the framework? I have found that it has already bitrotted at several places.